docs/guide/result.md

```diff
@@ -110,7 +110,7 @@ An example of output:
 ];
 ```
 
-Note that the result is always a 2 dimensional array with attribute names as keys at the first nesting level. This means
+Note that the result is always a 2-dimensional array with attribute names as keys at the first nesting level. This means
 that further nesting of attributes is not supported. Returning to the previous example, when `name` and `email` belong 
 to a `user` attribute, the output will be:
```

Comment:
>  This means
that further nesting of attributes is not supported.

is this also the case when using dot notation for nested attributes?

Reply:
Reading further, it seems that `getErrorMessagesIndexedByPath()` is doing what is not supported by this function, a hint about using the other method for it would be nice.